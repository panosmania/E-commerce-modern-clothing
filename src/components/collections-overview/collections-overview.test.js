import { shallow } from "enzyme";
import React from "react";
import toJson from "enzyme-to-json";

import { collectionsOverview } from "./collections-overview.component";

describe("collectionsOverview component", () => {
  const wrapper = shallow(<collectionsOverview collections={[]} />);
  it("should render CollectionsOverview component", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
