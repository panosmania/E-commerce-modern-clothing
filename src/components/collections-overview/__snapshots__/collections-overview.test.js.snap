// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`collectionsOverview component should render CollectionsOverview component 1`] = `
<collectionsOverview
  collections={Array []}
/>
`;
